src/Symfony/Component/Security/Guard/GuardAuthenticatorHandler.php

```diff
@@ -54,10 +54,8 @@ public function __construct(TokenStorageInterface $tokenStorage, EventDispatcher
      *
      * @param string $providerKey The name of the provider/firewall being used for authentication
      */
-    public function authenticateWithToken(TokenInterface $token, Request $request/*, string $providerKey */)
+    public function authenticateWithToken(TokenInterface $token, Request $request, string $providerKey = null)
```

Comment:
this is also a BC break

Reply:
This arguments is commented since Symfony 3.4, and the class passed final since 4.0, this particular evolution was skipped in the newest version, you are right it's need to be made in 5.0, not now